asv_bench/benchmarks/alignment.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+
+import xarray as xr
+
+from . import parameterized, requires_dask
+
+ntime = 365 * 30
+nx = 50
+ny = 50
+
+rs = np.random.RandomState(0)
```

Comment:
```suggestion
rng = np.random.default_rng(0)
```
Randomstate is not recommended anymore.